### PR TITLE
1 build frontend codebase

### PR DESCRIPTION
**Pull request by nicolardi**
Please blame me to show how git works during the lesson

### CODE REVIEW

File: features/suites/Auth/login.feature [hunk 1]
```diff
@@ -13,5 +13,29 @@ Feature: Login into the application
     Then I should be on "/home" page
     And I should see "Welcome"
 
-  Scenario: As a developer I wanna create a merge issue
```
Comment (rdeleo) [inline features/suites/Auth/login.feature]: What's that? Why did you remove the merge issue? I liked that!

**Review by DuffmanCC — APPROVED** [review]

**Review by rdeleo — APPROVED** [review]
You know what? Since Carlos approved I will do it as well!

**Review by rdeleo — APPROVED** [review]

File: features/suites/Auth/login.feature (hunk 1 above)
Comment (linhunix) [reply features/suites/Auth/login.feature]: i don't accept because i don use phpstorm

**Review by linhunix — CHANGES_REQUESTED** [review]
use netbeans

**Review by francescobianco — APPROVED** [review]
For me is ok!!! Please approve it!! We need to deploy ASAP

**Review by 39ro — APPROVED** [review]
Yeah now looks better!

**Review by linhunix — APPROVED** [review]

**Review by ShayAlam — APPROVED** [review]
Looks good

File: features/suites/Auth/login.feature (hunk 1 above)
Comment (39ro) [reply features/suites/Auth/login.feature]: Please send me a phpstorm license, I send nud3s.
